### PR TITLE
Feature/second nav

### LINKED ISSUE #61: Add second header/navbar that displays the FAQ Blog and Newsletter
Description:

Reporter:
Stephan Ellenberger 

Trello link:
https://trello.com/c/5x1bpAaB/165-add-second-header-navbar-that-displays-the-faq-blog-and-newsletter

Comments:

### PR DESCRIPTION
**Pull request by tunangst**
# Description

Please include a brief summary of what the issue is and what has been done to solve it?

Issue: Needed an extra top navbar. Add a few links to the added navbar.
Fix: I added the extra nav bar the same height as the original (may be a little too tall for the site, wouldn't mind an opinion on this). All links seem to work besides the newsletter link seems broken ( it is hitting /newsletter).

Fixes #61 

# Checklist:

- [✔️] I have commented my code, particularly in hard-to-understand areas
- [✔️] I have reviewed the 'files changed' tab on github to ensure all changes are expected
- [✔️] I have added someone to review the pr

### CODE REVIEW

**Comment by tunangst** [comment]
Also nested the styling as we like.
Pardon my formatter adding extra space so I can see :] (files changed)

File: src/components/top_nav/TopNav.module.scss [hunk 1]
```diff
@@ -1,131 +1,211 @@
...
+            align-items: center;
+            width: 100%;
+            justify-content: flex-end;
+            display: flex;
+            list-style: none;
+            box-sizing: border-box;
+
+            @media (max-width: $medium) {
+                display: none;
+            }
```
Comment (stellenberger) [inline src/components/top_nav/TopNav.module.scss:32]: If you look at the topNavContainer, the display has already been set to none for this screen size so I dont think you need to put it here, too?

File: src/components/top_nav/TopNav.module.scss [hunk 2]
```diff
@@ -1,131 +1,211 @@
...
+
+            li {
+                margin-left: calc(0.03 * 100vw);
+                @media (max-width: $medium) {
+                    padding: 6px;
+                }
```
Comment (stellenberger) [inline src/components/top_nav/TopNav.module.scss:38]: For this screen size the container has been set to display: none. Or is this for another purpose which I've missed?? 😄

File: src/components/top_nav/TopNav.module.scss [hunk 3]
```diff
@@ -1,131 +1,211 @@
...
+                    }
+
+                    @media (max-width: $medium) {
+                        display: none;
+                    }
+
+                    li {
+                        @media (max-width: $medium) {
+                            padding: 6px;
+                        }
```
Comment (stellenberger) [inline src/components/top_nav/TopNav.module.scss:193]: Same as above

**Review by stellenberger — COMMENTED** [review]
Looks fantastic thank you so much! 🎉 

Just a few comments that I made, although please forgive me if I have overseen something that it was in place for. There were some media queries which didnt seem to do anything as their parent elements were set to display: none. This is a tiny thing and the rest looks absolutely beautiful 🔥  🌶️ 

One last thing if you wanted to do, navRow and topNavRow are identical except for one media query and the justify content. If you replace both justify content with flex-end and use padding to space the items, we could merge it into one to honour DRY?

File: src/components/top_nav/TopNav.module.scss (hunk 2 above)
Comment (tunangst) [reply src/components/top_nav/TopNav.module.scss]: No, you were 100% right, this was an oversight from my copying code from previous dev :]

**Comment by tunangst** [comment]
Sure, of course we can do that, I was trying not to rework the previous dev's code. Now that you pointed it out, It would most likely benefit our future team if I reworked that whole component. I will work on this hopefully a little this wednesday +.

**Pull request closed without merging**